docs/data_sharing_guide.md

```diff
@@ -0,0 +1,338 @@
+# Data Sharing Guide: CDM JupyterHub
+
+## Overview
+
+The BER DataLake JupyterHub platform provides data sharing capabilities that allow users to share their datasets with other users and groups. 
+
+All data governance functions are **✨ automatically imported** in every notebook via `startup.py` - no manual imports needed! This guide shows you how to use these pre-loaded functions for seamless data sharing and collaboration.
+
+## Key Concepts
+
+### Data Organization
+
+All user data in BER DataLake is organized under personal namespaces:
+
+- **General Data Storage**: `s3a://cdm-lake/users-general-warehouse/{username}/`
+- **SQL Warehouse**: `s3a://cdm-lake/users-sql-warehouse/{username}/`
```

Comment:
Does there need to be a discussion about the global Hive namespace vs. the individual SQL warehouse namespaces? I can see that causing confusion. I'm not actually sure what happens if a table name exists in hive, but there's no table in the user's sql warehouse, for example - there's probably a number of weird things users could encounter

Reply:
Ok, if you're planning to fix the global namespace stuff in the near future that's fine

Reply:
Yea - just to be sure we are talking about the same thing. I am planning to work with science team and move all existing tables to users individual SQL warehouse.

Reply:
created ticket https://kbase-jira.atlassian.net/browse/CDM-314 for this.